### ISSUE TITLE
ENL problem

### ISSUE THREAD

**Issue by deltafox44** [weglide/bugtracker#121]
ENL threshold seems too low, opening the window leads to false engine run detection
See https://www.weglide.org/flight/43170 at 14:07 & 14:10 (I had no problem with the same flight on OLC)
(I edited the flight to correct the soaring window times)

**Comment by samuel-git** [comment]
Thanks for reporting, ENL level calibration is notoriously difficult and is (at the moment) a tradeoff between false-positives and false-negatives. We will take some days especially fort that and are planing to do so.

**Comment by deltafox44** [comment]
At the contrary, in this flight https://www.weglide.org/flight/29580 it seems that there is a non detected engine run (see after 18:05). May be due to the type declared as a pure glider (Duo Discus) instead of the motorglider it seems to be ([Arcus](https://www.weglide.org/flight/29601))

**Comment by samuel-git** [comment]
@deltafox44 yes, when an pure glider is selected, we do not show/use the ENL data as a motor run is "impossible". Selecting a motorized glider will update the scoring windows.

**Comment by deltafox44** [comment]
I understand that. The problem is when the glider type has not been recognized from the logger file. See for instance : https://www.weglide.org/flight/64296
HFGTYGLIDERTYPE:ASH_26_E
HFGIDGLIDERID:D_KIDF
in the file is interpreted as "LS 8 D-2978 KS" !!!
Also, in [my list of flights](https://www.weglide.org/flight?user=Denis%2520Flament,4019&order_by=-scoring_date) , I don't know why the type many times happens to be "ASH 26" instead of ""ASH 26 E" (with ASH_26E or ASH 26E in the file) and sometimes not.
I would say that if ENL is recorded in the file and an engine run is detected, and the type is a pure glider, the most likely is that the type is not correct ! Thus why not use it...?

**Comment by flyingjoe** [comment]
That's an interesting edge case you came across :smile: 
We only use the glider type in the igc file once and after that always use the glider that was chosen by the pilot for that particular glider. 
Newer FR use 3 character long unique ids, but the LX5000 seems to use 5: AFIL02336FLIGHT:1 
This leads to your LX5000 being seen as the same as all other LX5000 with an id beginning with "023".

The second problem is the result of uploading files without changing the glider immediately. This is annoying for the initial upload of many flights, but shouldn't be a problem in daily use. 

Pilots flying a glider without an engine usually don't check their scoring windows as closely as those with an engine. 
Enabling ENL for everyone would lead to a lot of incorrectly set scoring windows.

**Comment by deltafox44** [comment]
Engine detection has been much improved since I began this thread - thanks. However, I got a problem two days ago on [this flight](url), where there is no "Task" tab despite the task was completed. 

I noticed that the soaring performance begins at 09:42:10 only, just after passing the start line (between 09:42:05 and 09:42:10), despite the engine was actually stopped between 09:41:57 and 09:42:01 (where the altitude is at its max and ENL has began to drop. Perhaps the time soaring performance determined by the algorithm should begin a few seconds before all noise stops...

Unfortunately, I did not noticed the problem sooner and two days after the flight it is no longer possible to edit the flight ("Edit period of scoring relevant entries already over.").

Also, the start point coordinates declared via a WeGlide Task are not correct - see screenshot, the start line should be centered on "La Roche sur Yon" airport but it is about 200m apart - since it is quite a different problem I open another thread for that. 

![image](https://user-images.githubusercontent.com/82087770/170719003-bfa92fc4-9cb3-49b4-ac9a-cd63f9183f01.png)
![image](https://user-images.githubusercontent.com/82087770/170719091-091a6525-a79b-44a7-8b7e-0beaf176a96e.png)
